### PR TITLE
Added Gmail Notification

### PR DESCRIPTION
**Pull request by andreaseide80**
I have added Gmail notifications with dependencies and config defaults.
If enabled and config is correct it sends email to recipient with changes or same notifications as to desktop.

### CODE REVIEW

**Comment by RaphCPV** [comment]
Don't forget to change the version number before merging :)

**Comment by marklagendijk** [comment]
@andreaseide80 thanks for your pull request.

Why would you want email notifications?

**Comment by andreaseide80** [comment]
I want email notifications as I'm not always in front of my computer, but I
have a android wear watch and get the emails on my phone and watch.
This way I get notified where I am. Without having to slide the app all the
time.

fre. 4. sep. 2020, 19:43 skrev Mark Lagendijk <notifications@github.com>:

**Comment by schklom** [comment]
I support this feature. Being able to get notification without being in front of a computer or connected to a specific app is great. My phone restricts apps in background. I already have to allow emails to come in constantly, and having to do that for Telegram as well would drain my battery that much faster.

**Comment by marklagendijk** [comment]
This can only be merged if it is made general for all email providers. There is no reason to make it gmail-specific.

File: lib/notifier.js [hunk 1]
```diff
@@ -4,18 +4,42 @@ const telegramBot = require("./telegram-bot");
...
+    port: 465,
+    auth: {
+       user: gmailoptions.gmail.username,
+       pass: gmailoptions.gmail.password
+    }
+});
+
+function cal(m) {
+  return m.calendar(null, {
+    lastDay: "[I går] HH:mm",
```
Comment (Plavit) [inline lib/notifier.js:28]: Why is this in swedish again? Will it be platform independent?

**Comment by marklagendijk** [comment]
I'm closing this PR, because it is inactive and needs major changes.

**Pull request closed without merging**